actions/docs-verifier/src/ActionRunner/Program.cs

```diff
@@ -20,9 +19,28 @@
     throw new InvalidOperationException($"The value of GITHUB_PR_NUMBER environment variable is not valid.");
 }
 
+static bool IsYmlOrMarkdownFile(PullRequestFile file) => Path.GetExtension(file.FileName) is ".yml" or ".md";
+
+static bool IsInWhatsNewDirectory(PullRequestFile file)
+{
+    string? whatsNewPath = WhatsNewConfigurationReader.GetWhatsNewPath();
+    if (whatsNewPath is { Length: > 0 })
+    {
+        // Example:
+        // file.FileName:   docs/whats-new/2021-03.md
+        // whatsNewPath:    docs/whats-new
+
+        return whatsNewPath.StartsWith(file.FileName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Should these be reversed?

```suggestion
        return file.FileName.StartsWith(whatsNewPath, StringComparison.OrdinalIgnoreCase);
```

Reply:
Admittedly, I'm going by inspection here....